### DISCUSSION TITLE
Test with Jest/Ava same import error (part 2)

### DISCUSSION THREAD

**Discussion by giovanni9793** [panva/jose#166] [Q&A] [answered]
As I said I read the threads about the discussion and the solutions proposed do NOT work for me. The problem is the same in ava, so both jest and ava have the same bug? If yes, you only suggest using ava for other reasons I suppose, so what could be a solution for my problem, since the other solutions don't work for me?

**Comment by panva** [comment, marked as answer]
@giovanni9793 please raise your Jest related concerns in #105 where you may actually get help from others. If you have the same issue with ava as you do with jest, it doesn't mean they both have the same bug/issue, it's just that maybe their underlying typescript loader is the same that has a missing support for the way the package's modules are setup. Others were successful in working around jest's deficiencies in #105 so that matter is closed for me and I cannot help further than to point you to #105 or referring you to jest's issue/support tracker.

**Comment by panva** [comment]
I'm done waiting for these ecosystems to catch up. jose v4.x will be coming out today/tomorrow with only top level named exports that solve all these tools' problems.
